Example/ViewController.swift

```diff
@@ -311,6 +313,7 @@ class ViewController: UIViewController {
     }
 
     func presentAndRemoveMapview(_ navigationViewController: NavigationViewController, completion: CompletionHandler?) {
+        navigationViewController.modalPresentationStyle = .fullScreen
```

Comment:
Should we add this line to the “[Configuration](https://github.com/mapbox/mapbox-navigation-ios/#configuration)” section of the readme and any corresponding example code?